Panic when consuming a (big) notification 2.0 backlog

I've consumed a Notification 2.0 backlog of ~ 150k messages via: `c8y notification2 subscriptions subscribe --name {my subscription name} --subscriber {subscriber name}`. 

After ~ 100k messages this panic got thrown:
```text
panic: concurrent write to websocket connection
 
goroutine 1 [running]:
github.com/gorilla/websocket.(*messageWriter).flushFrame(0xc0007845a0, 0x1, {0x0?, 0x0?, 0x0?})
	github.com/gorilla/websocket@v1.5.1/conn.go:632 +0x4b8
github.com/gorilla/websocket.(*messageWriter).Close(0xa3d7d8?)
	github.com/gorilla/websocket@v1.5.1/conn.go:746 +0x35
github.com/gorilla/websocket.(*Conn).beginMessage(0xc000737ce0, 0xc00081e840, 0x1)
	github.com/gorilla/websocket@v1.5.1/conn.go:493 +0x47
github.com/gorilla/websocket.(*Conn).NextWriter(0xc000737ce0, 0x1)
	github.com/gorilla/websocket@v1.5.1/conn.go:535 +0x3f
github.com/gorilla/websocket.(*Conn).WriteMessage(0xc18e9a9c60b784d0?, 0x29f87f4050?, {0xc0002bc000, 0x10, 0x1000})
	github.com/gorilla/websocket@v1.5.1/conn.go:788 +0x137
github.com/reubenmiller/go-c8y/pkg/c8y/notification2.(*Notification2Client).SendMessageAck(0xc000134b00, {0xc0002bc000, 0x10, 0x1000})
	github.com/reubenmiller/go-c8y@v0.16.0/pkg/c8y/notification2/notification2.go:388 +0x113
github.com/reubenmiller/go-c8y-cli/v2/pkg/cmd/notification2/subscriptions/subscribe.(*SubscribeCmd).RunE(0xc0002e1cb0, 0xc0004cbb08, {0x0?, 0x0?, 0x0?})
	github.com/reubenmiller/go-c8y-cli/v2/pkg/cmd/notification2/subscriptions/subscribe/subscribe.manual.go:175 +0x6fc
github.com/reubenmiller/go-c8y-cli/v2/pkg/cmd/notification2/subscriptions/subscribe.NewSubscribeCmd.NewSubCommand.func4(0xc0004cbb08, {0xc0005bfb00, 0x0, 0x4})
	gith:55 +0x62
github.com/spf13/cobra.(*Command).execute(0xc0004cbb08, {0xc0005bfac0, 0x4, 0x4})
	github.com/spf13/cobra@v1.8.0/command.go:983 +0xaca
github.com/spf13/cobra.(*Command).ExecuteC(0xc0002e8308)
	github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	github.com/spf13/cobra@v1.8.0/command.go:1039
github.com/reubenmiller/go-c8y-cli/v2/pkg/cmd.MainRun()
	github.com/reubenmiller/go-c8y-cli/v2/pkg/cmd/cmd.go:124 +0x1d0
main.main()
	./main.go:8 +0xf
```

When restarting the subscription, the rest of the backlog was consumed fine. Not sure if there is something to "fix", but I guess the panic should be handled to e.g. show a proper error message and maybe even retry after some time?

--- a/go.mod
+++ b/go.mod
@@ -24,7 +24,7 @@ require (
 	github.com/olekukonko/ts v0.0.0-20171002115256-78ecb04241c0
 	github.com/pkg/errors v0.9.1
 	github.com/pquerna/otp v1.4.0
-	github.com/reubenmiller/go-c8y v0.16.0
+	github.com/reubenmiller/go-c8y v0.19.1
 	github.com/santhosh-tekuri/jsonschema/v5 v5.3.1
 	github.com/sergi/go-diff v1.3.1 // indirect
 	github.com/sethvargo/go-password v0.3.0

